.travis.yml

```diff
@@ -1,10 +1,32 @@
 language: node_js
-os: 
-  - windows
-  - osx
-  - linux
 node_js:
   - "lts/*"
 notifications:
   email: false
-  
+
+# Default phases (can be overridden in jobs)
+install: npm ci
+script:
```

Comment:
should lint be a stage instead of a separate script?

Reply:
So, I thought about this originally, but it just seemed really unnecessary to spin up another VM just for the linting.  The real value of separating the publish and test stages is that we prevent the publish from running at all if the test stage fails.  AFIK we do not really need to prevent the tests from running if the linting fails, so we would not be gaining too much...

Reply:
It just feels a bit scattered to have both the script array and the jobs